### ISSUE TITLE
Call to undefined method League\Csv\Reader::setHeaderOffset()

### ISSUE THREAD

**Issue by joao-gsneto** [thephpleague/csv#226]
## Issue summary
Call to undefined method setHeaderOffset when try to reproduce the readme documentation on a file

_(Please explain in plain english your issue/feature request)_
```php
require 'vendor/autoload.php';


use League\Csv\Reader;

$csv = Reader::createFromPath('../dados/Dados Dragao/Dados_Dragao.csv')
	->setHeaderOffset(0);
```

### System informations
php 7.1 on Windows 10

### Expected result
Works (Because this is the official documentation example)

### Actual result
Fatal error

**Comment by nyamsprod** [comment]
The code you are using is for League\Csv 9.+ which is documented on http://csv.thephpleague.com/9.0
The current stable version is League\Csv 8.x which is documented on http://csv.thephpleague.com/8.0 and whose public API does not contains the `Reader::setHeaderOffset` method.
With the next major version being release dsoon the documentation website is  being updated for that released

**Comment by joao-gsneto** [comment]
Thanks @nyamsprod  but installing via composer

```
composer require league/csv
```
by default installs the 8.x version and this make the confusion.

Install with
```
composer require league/csv:9.x-dev
```
solves this problem.

**Comment by Kozin-br** [comment]
> The code you are using is for League\Csv 9.+ which is documented on http://csv.thephpleague.com/9.0
> The current stable version is League\Csv 8.x which is documented on http://csv.thephpleague.com/8.0 and whose public API does not contains the `Reader::setHeaderOffset` method.
> With the next major version being released soon the documentation website has been updated for that released

Why your ofiicial documentation contains dev code? i think must not be there...
